### PR TITLE
No telegram rate updates when orderbook is enabled

### PR DESCRIPTION
**Pull request by xmatthias**
## Summary
get_ticker is not updating when orderbook is enabled since it's never called.
extacted the get_ticker logic (using [bid] ...) to a separate function which is called from RPC.

Solve the issue: #1658

## Quick changelog

- Reenable updates when orderbook asks  is on

*note*: before using this, read the outcome of #1667  - I may have discovered a bug in the orderbook ask logic.

### CODE REVIEW

File: freqtrade/freqtradebot.py [hunk 1]
```diff
@@ -579,6 +579,25 @@ def get_real_amount(self, trade: Trade, order: Dict) -> float:
...
+        for BitMex (has no bid/ask in get_ticker)
+        or remain static in any other case since it's not updating.
+        :return: Bid rate
+        """
+        config_ask_strategy = self.config.get('ask_strategy', {})
+        if config_ask_strategy.get('use_order_book', False):
+            logger.debug('Using order book to get sell rate')
+
+            order_book = self.exchange.get_order_book(pair, 1)
+            rate = order_book['asks'][0][0]
```
Comment (mishaker) [inline freqtrade/freqtradebot.py]: Ok, this is just a refactoring (sell rate fetch wise). but I think we have to convert that "asks" to "bids". but we can do it in a separate PR ... or here ... either way.

Comment (xmatthias) [reply freqtrade/freqtradebot.py]: Good point!
